packages/e2e-test-utils/src/activate-theme.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { switchUserToAdmin } from './switch-user-to-admin';
+import { switchUserToTest } from './switch-user-to-test';
+import { visitAdminPage } from './visit-admin-page';
+
+/**
+ * Activates an installed theme.
+ *
+ * @param {string} slug Theme slug.
+ */
+export async function activateTheme( slug ) {
```

Comment:
Do you think we need functions to deactivate and uninstall themes as well? (like plugins).

Reply:
I can see uninstalling themes being useful. I'll add that. As for deactivating, deactivating a theme just means switching to a different theme, so I think that's covered by `activateTheme`.

Reply:
Done in aef01296553bf7b3ac75193ada82d5f4b6996b5a